README.md

```diff
@@ -1,148 +1,28 @@
-# Istio Authentication
+# Istio Security
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/istio/auth)](https://goreportcard.com/report/github.com/istio/auth)
 [![codecov](https://codecov.io/gh/istio/auth/branch/master/graph/badge.svg)](https://codecov.io/gh/istio/auth)
 
```

Comment:
Do you think we need more general contents to redirect to other pages, such as architecture (authentication.md), contributing (contributing.md) and testing (I can add this page)?

Reply:
Sure, that would be great.